ios/HMSView.swift

```diff
@@ -89,6 +99,27 @@ class HmssdkDisplayView: UIView {
         }
     }
 
+    @objc func captureHmsView( _ requestId: NSNumber) {
+        guard let onDataReturnedUnwrapped = onDataReturned else {
+            print(#function, "Can't send any data to JS side, `onDataReturned` is nil!")
+            return
+        }
+
+        guard let image = videoView.captureSnapshot() else {
+            print(#function, "Could not capture snapshot of HMSVideoView")
```

Comment:
should we remove this print statement? @ygit

Reply:
No, should not remove printing error logs